db/schema.rb

```diff
@@ -61,7 +63,7 @@
     t.string "type", default: "Team::Shared", null: false
     t.integer "personal_owner_id"
     t.integer "recrypt_state", default: 1, null: false
-    t.string "encryption_algorithm"
+    t.string "encryption_algorithm", default: "AES256", null: false
```

Comment:
no default ... make sure you're checking in the right schema.rb
check this file out ... then run db:migrate ... this should create an up to date version